src/unabto/unabto_types.h

```diff
@@ -6,7 +6,8 @@
 enum {
     FINGERPRINT_LENGTH = 16,
     PSK_ID_LENGTH = 16,
-    PSK_LENGTH = 16
+    PSK_LENGTH = 16,
+    FCM_TOKEN_LENGTH = 255
```

Comment:
The FCM token is not a part of the core, why should the define be in the core?

Reply:
Records in persistence.bin storage will always have a buffer of FCM_TOKEN_LENGTH inside them. The difference is whether this buffer gets loaded into transient memory or not.

Reply:
FCM_TOKEN_LENGTH and the token structs have been moved to `fp_acl.h`